Tiltfile

```diff
@@ -18,6 +20,13 @@ docker_build('tilt-site', 'src', dockerfile='site.dockerfile',
 
 docker_build('docs-site', '.', dockerfile='docs.dockerfile',
              live_update = [
+               # we don't care about this being on the container but we don't want it to trigger a full rebuild,
+               # just toss it somewhere so it's accounted for by a LiveUpdate step
+               # This is hacky. Right now a change to api.py kicks off TWO container updates, one
+               # to sync api.py (which we don't care about), and one when the local('make api') call
+               # above changes docs/_includes/api.html -- but it's still faster than a full rebuild ¯\_(ツ)_/¯
+               sync('./api/api.py', '/tmp/api.py'),
```

Comment:
I feel like this comment points to a bigger problem with the api - full-rebuild-on is kind of unhelpful most of the time, the more common case is an ignore param

Have you considered putting api.py into tiltignore?

Reply:
yes I tried that -- the result is that changes to api.py don't kick off tiltfile re-execution :-/